### ISSUE TITLE
Meta tab needs to go.

### ISSUE THREAD

**Issue by Artefact2** [osmium-org/osmium#74]
It's very space efficient, and does make sense, but the world is not ready for it. People keep asking (and they're right, it's not obvious at first look) how to delete or export lodaouts, this needs to be more self-explanatory.

Feedback on how to do this would be appreciated.

**Comment by jboning** [comment]
For the sake of easy discussion, listing out the things on the meta tab:

#### Share
- link to loadout
- link to this revision
- snippets for linking in markdown/bbcode/html

#### Export
- most formats open a model with a textarea containing the exported loadout (preselected)
- in-game DNA calls the IGB JS hook to show an loadout

#### Permissions
- who can view
- who can edit
- whether it's publicly searchable
- whether it's subject to public moderation

#### Actions
- edit
- delete
- add to favorites
- browse loadouts for this ship
- browse loadouts from this user
- zkillboard activity for this ship

**Comment by jboning** [comment]
I think the top thing to tackle is exporting--it's by far the thing I hear the most questions about, among the people I've aimed at Osmium.

I think something like zKillboard does for their external/export/social sections would work nicely. I'd probably put this directly to the right of the loadout name, or perhaps on in the top right corrner (to the right of the tags). A possible modification would be to have a split button, with the default action being to open the EFT modal directly, and other formats hidden in the dropdown. (I think that this UI could replace the export UI in the editor UI, too--format dropdown + export button is pretty clunky.)

The big drawback here is that we lose the space to explain what the various formats are. Maybe the Dropdown could include a "help" option? Honestly I don't think that the options besides EFT/in-game DNA/DNA provide a lot of value :/ I'm kind of curious what the stats are on usage, my suspicion is that it's like 90% EFT, 9% in-game DNA, 1% everything else.

**Comment by jboning** [comment]
For sharing: my first idea would be to ditch this section entirely and just trust that users know how to copy URLs and write whatever format they're working in. With other export UI in place, I'd probably change the loadout title to be a link to the loadout (much like the way reddit self.* posts work).

I'm not sure about linking to a particular revision. Maybe I'd just let the history tab handle this? Or, in the credits section, "Revision #n" could be a permalink to the latest revision.

**Comment by jboning** [comment]
Actions: There's already an edit button in the tab bar. I'd suggest putting delete and flag there as well. If we're worried about screen real estate, pencil/trashcan/flag icons are very representative (and accessibility can be maintained by alt-text). A star for "favorite" is also pretty iconic; I'm not sure where this would go, since the tab bar doesn't seem like the right place. 

"Browse loadouts from [user]" can be tossed without much loss--the link to the author's name in the credits section does just as well.

I really don't have a good answer for "Browse all [ship] loadouts" and "[ship] activity on zKillboard". Will keep pondering....

**Comment by jboning** [comment]
I'm also pretty low on ideas for what to do with Permissions. Though, if all of the other stuff gets moved out, we can rename the tab "Perms" and it will be descriptive at least :)

**Comment by matticusrex** [comment]
Hi, I opened #98 yesterday and I see the problem now.  I think if you just changed the tab name to "Share" or "Export" or really any number of things other than "Meta" it would be a lot clearer what the purpose of that tab is.  As a first time visitor yesterday, I was looking for an export option, but failed to click the "Meta" tab because I associated it with the EVE term for module ranks.  It's really not a very descriptive word in the first place.  Thank you for your consideration, and for your time on this excellent project.
